packages/components/text/src/text.ts

```diff
@@ -0,0 +1,39 @@
+import { buildProps } from '@element-plus/utils'
+import { componentSizes } from '@element-plus/constants'
+import type Text from './text.vue'
+
+import type { ExtractPropTypes } from 'vue'
+
+export const textProps = buildProps({
+  /**
+   * @description text type
+   */
+  type: {
+    type: String,
+    values: ['primary', 'success', 'info', 'warning', 'danger', ''],
+    default: '',
+  },
+  /**
+   * @description text size
+   */
+  size: {
+    type: String,
+    values: componentSizes,
+    default: '',
+  },
+  /**
+   * @description render ellipsis
+   */
+  truncated: {
+    type: Boolean,
+  },
+  /**
+   * @description override the element
+   */
+  as: {
```

Comment:
as -> tag ?

refer [tag attributes of Row and Col](https://element-plus.org/en-US/component/layout.html#row-attributes)

Reply:
as -> tag modified, also modify the docs and unit test.